### PR TITLE
git-sync --force will sync without interaction

### PR DESCRIPTION
**Pull request by timfeirg**
this further improves use speed to an already epic command.

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Would you please update the doc under `man/` too?

**Comment by timfeirg** [comment]
I've added docs for man/git-sync.md, could you help me with the ronn auto generation part? I haven't figured out how to make that happen. maybe add some dev docs? or even better, include this as a pre-commit step.

@spacewander

File: man/git-sync.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ git-sync(1) -- Sync local branch with remote branch
 
 ## SYNOPSIS
 
-  `git sync` [--soft] [ &lt;remote&gt; &lt;branch&gt; ]
+  `git sync` [--soft] [--force] [ &lt;remote&gt; &lt;branch&gt; ]
```
Comment (spacewander) [inline man/git-sync.md:6]: Let's generate other doc like https://github.com/tj/git-extras/tree/master/man#description

Comment (spacewander) [reply man/git-sync.md:6]: > even better, include this as a pre-commit step

Any pull request to improve the doc generation is welcome!

**Comment by timfeirg** [comment]
wasn't paying attention to CONTRIBUTING.md when creating this PR, now docs have been generated.

**Review by spacewander — APPROVED** [review]

**Comment by timfeirg** [comment]
I have no write power, is this going to merge?

**Comment by spacewander** [comment]
@timfeirg 
Done. Thanks for your contribution.

**Pull request merged**